### ISSUE TITLE
A11y_Azure Tools for VS Code [BM]_Select Sunscriptions_Non-Text Contrast: Color contrast ratio for "list item focus" with it's background is 1.1:1 which is less than 3:1

### ISSUE THREAD

**Issue by Accessibility56** [microsoft/vscode-azurefunctions#2680]
[Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

### **GitHubTags:**
#Azure Tools for VS Code; #A11ySev2; #A11yTCS; #BM-Azure Tools for VS Code-Win-Feb2021; #DesktopApp; #Non-Text Contrast; #WCAG1.4.11; #Win10;

**OS Details:**
Windows 10 Version 20H2 (OS Build 19042.685)

### **Environment Details:**
Application Name: Azure Tools for VS Code
Version: 1.53.1
Commit: 5d424b828ada08e1eb9f95d6cb41120234ef57c7
Date: 2021-02-08T23:29:42.785Z
Electron: 11.2.1
Chrome: 87.0.4280.141
Node.js: 12.18.3
V8: 8.7.220.31-electron.0
OS: Windows_NT x64 10.0.19042
Azure Function Version: 1.2.1

### Repro Steps:

1. Open Visual studio code
2. Tab till extension button and press enter on it.
3. Search for "Azure Functions" and select it.
4. Tab till "Install" button and press enter on it.
5. Tab till "Azure" button from left side pane and press enter on it.
6. Default screen will get open.
7. Tab to Tree view and use down arrow to navigate to tree items.
8. Tab to "Select Subscriptions" and press enter on it.
9. Navigate to list items using down arrow
10. Verify Contrast ratio for "List item focus" with it's background

### Actual Result:
Color contrast ratio for "list item focus" with it's background is 1.1:1 which is less than 3:1

### Expected Result:
Color contrast ratio for "list item focus" with it's background should be more than 3:1

### User Impact:
Low vision users will find difficulty in viewing the content and to perform operations if color contrast ratio is not properly maintained

### Attachment for Reference:
![2680_Non Text Contrast_Filter](https://user-images.githubusercontent.com/38523250/108210373-7722ef80-7151-11eb-8ebc-8816d71d4f64.png)

**Comment by ejizba** [comment]
Closing in favor of upstream issue https://github.com/microsoft/vscode/issues/121493

**Comment by Accessibility56** [comment]
We will take action once the external bug is closed and fix is available in latest VS build
